conversion/conversion.go

```diff
@@ -417,6 +417,76 @@ func getSeekable(f *os.File) (*os.File, int64, error) {
 	return fcopy, n, nil
 }
 
+// VerifyDb checks whether the db exists and if it does, verifies if the schema is what we currently support.
+func VerifyDb(project, instance, dbName string) (dbExists bool, err error) {
+	ctx := context.Background()
```

Comment:
Consider cleaning up some of this in a follow up PR as following: 
1. Create dbURI in cmd.CommandLine and pass that around to all these helper functions instead of passing project, instance, dbName parameters (unless a specific function requires these parameters separately); 
2. Pass ctx and adminClient from cmd.CommandLine to these functions